src/views/Dashboard/Lab/Registration/Services/index.vue

```diff
@@ -298,74 +298,79 @@ export default {
 
     serviceCategoryRules() {
       return [
-        val => !!val || 'This field is required'
+        val => !!val || 'This field is required.'
       ]
     },
 
     biologicalTypeRules() {
       return [
-        val => !!val || 'This field is required'
+        val => !!val || 'This field is required.'
       ]
     },
 
     serviceNameRules() {
       return [
-        val => !!val || 'This field is required',
-        val => (val && val.length <= 50) || 'This field only allows 50 characters'
+        val => !!val || 'This field is required.',
+        val => (val && val.length <= 50) || 'This field only allows 50 characters',
+        val => (val && /^[A-Za-z? ]?[A-Za-z0-9? ]+$/.test(val) || "This field can only contain English alphabet")
       ]
     },
 
     curencyTypeRules() {
       return [
-        val => !!val || 'Currency Type is required'
+        val => !!val || 'This field is required.'
```

Comment:
SonarCloud recommended that all `This field is required.` rules be put into one rule. Just name it `fieldRequiredRule.` If more rules need to be added in the future, you can separate them into their own rules.